src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  FC, memo, useContext, useState,
+} from 'react';
+
+import cn from 'classnames';
+
+import { SelectUserIdContext } from '../../contexts/SelectUserIdContext';
+
+type Props = Omit<Todo, 'id'>;
+
+export const TodoItem: FC<Props> = memo(({
+  userId, completed, title,
+}) => {
+  const { selectedUserId, setSelectedUserId } = useContext(SelectUserIdContext);
+  const [checked, setChecked] = useState(completed);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked(event.target.checked);
+  };
```

Comment:
Your solution just checks the Todo. But it doesn't save anything. There is an API link to update the Todo.  But I don't know if it works. If it doesn't - may be you could use readonly. I think you can skip it.
![image](https://user-images.githubusercontent.com/47161502/163154816-49fd566d-a3f4-4392-9e9a-c53b082c1259.png)